subt/docker/robotika/ros_proxy_node.cc

```diff
@@ -336,6 +349,44 @@ not available.");
 }
 
 
+void Controller::sendLogPart()
+{
+  if (!this->started)
+    return;
+  // Test empty log
+  // based on Nate example:
+  //   https://bitbucket.org/osrf/subt_seed/branch/log_sensor_msgs#diff
+  std_msgs::String log;
+  std::ostringstream stream;
+  ros::Time currentTime = ros::Time::now();
+  stream << currentTime.sec + currentTime.nsec*1e-9 << ": Hello " << this->name;
+  log.data = stream.str().c_str();
```

Comment:
We know `std::string` can contain zeros - is the same true for `std_msgs::String`?

Reply:
Yes, data of `std_msgs::String` is `std::string`:
https://docs.ros.org/api/std_msgs/html/msg/String.html
(and it works)